docs/bold-bi-auto-deployment.md

```diff
@@ -0,0 +1,982 @@
+              BOLD_SERVICES_BRANDING_LOGIN_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png"
+              
+              # This is an email logo, and the preferred image size is 200 x 40 pixels.
+              BOLD_SERVICES_BRANDING_EMAIL_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png"
+              
+              # This is a favicon, and the preferred image size is 40 x 40 pixels.
+              BOLD_SERVICES_BRANDING_FAVICON: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png "
+              
+              # This is powered by the logo, and the preferred size is 100 x 25 pixels.
+              BOLD_SERVICES_BRANDING_FOOTER_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png "
+              
+              # This is organization name.
+              BOLD_SERVICES_SITE_NAME: "Autodeployment"
+              
+              # This is site identifier, and it will be the part of the application URL.
+              BOLD_SERVICES_SITE_IDENTIFIER: "branding"
+
+              # Note: If you want to use the custom branding, provide the value for all branding variables 
+              # If all variable values are given, application will use the branding images,
+              # otherwise it will take the default logos.
+            ---     
+        </details>
+
+           
+    * If you need to use **Bing Map** widget feature, enter value for `widget_bing_map_enable` environment variable as `true` and API key value for `widget_bing_map_api_key`.
+       
+       ![Enable-Bingmap](images/deploy/aks/bingmap_enable.png)
+
+6. If you have a DNS to map with the application, then you can continue with the following steps, else skip to **Step 11**. 
+
+7. Uncomment the host value and replace your DNS hostname with `example.com` in deploy_aks.yaml file in line **1442**.
+
+      ![DNS](images/deploy/aks/ingress_http_yaml.png)
+
+8. If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, follow the below step or you can skip to **Step 11**.
+
+9.  Run the following command to create a TLS secret with your SSL certificate.
+
+```sh
+kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+```
+
+10. Now, uncomment the `tls` section and replace your DNS hostname with `example.com` in ingress spec and save the file.
+
+       ![ingress DNS](images/deploy/aks/ingress_yaml.png)
+
+11. Now, run the following command to get the ingress IP address.
+
+      ```sh
+      kubectl get svc -n ingress-nginx
+      ```
+      Repeat the above command till you get the IP address in EXTERNAL-IP tab as shown in the following image. 
+      ![Ingress Address](images/deploy/aks/ingress_address.png) 
+
+12. Note the EXTERNAL-IP address and map it with your DNS, if you have added the DNS in **deploy_aks.yaml** file. If you do not have the DNS and want to use the application, then you can use the EXTERNAL-IP address.
```

Comment:
Note the EXTERNAL-IP address and map it to your DNS if you have added it in the **deploy_aks.yaml** file. If you do not have a DNS, use the EXTERNAL-IP address to access the application.

Reply:
concern addressed